Userland/Applications/Assistant/Providers.cpp

```diff
@@ -41,12 +44,25 @@ void FileResult::activate() const
     Desktop::Launcher::open(URL::create_with_file_protocol(title()));
 }
 
-void AppProvider::query(String const& query, Function<void(Vector<NonnullRefPtr<Result>>)> on_complete)
+void TerminalResult::activate() const
 {
-    if (query.starts_with("="))
+    pid_t pid;
+    char const* argv[] = { "Terminal", "-e", subtitle().characters(), nullptr };
```

Comment:
So the convention for the calculator provider is to have the title be the 'primary' action and the subtitle is used to display extra information as to what will happen. Would you be willing to switch this to have the subtitle give the extra info and the title is used as the string to run in the Terminal?

Reply:
Sure thing - looks like this now:
![term](https://user-images.githubusercontent.com/5600524/124355061-4792f200-dbdd-11eb-90a6-90d752caa3be.png)